### PR TITLE
front: select text overflow

### LINKED ISSUE #626: select: placeholder and value text overflow
### What happened?

Image

### What did you expect to happen?

The best approach would be to truncate the text with ‘…’ and add some padding.

### How can we reproduce it (as minimally and precisely as possible)?

Reduce the size of the select or use a very long text

### On which environments the bug occurs?

_No response_

### On which browser the bug occurs?

Firefox, Google Chrome

### PR DESCRIPTION
**Pull request by Wadjetz**
fix #626 
Capture d’écran 2024-10-09 à 16 29 57

### CODE REVIEW

**Review by achrafmohye — APPROVED** [review]
LGTM and tested :)

**Comment by thibautsailly** [comment]
Could you remove 8px to the label's width? It will give more room to the vertical arrows on their left side, and will make it easier for users to spot the select component at a glance when eye scanning a form. Thanks!

File: ui-core/src/styles/select.css [hunk 1]
```diff
@@ -6,6 +6,10 @@
     0px 2px 4px rgba(0, 0, 0, 0.1);
   height: 2.5rem;
   padding-left: 0.813rem;
+  padding-right: 1.875rem;
```
Comment (SharglutDev) [inline ui-core/src/styles/select.css]: We could win one line of code

```suggestion
  padding-inline: 0.813rem 1.875rem;
```

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested. Should we also add a `title` attribute @thibautsailly ?

**Comment by thibautsailly** [comment]
> Lgtm and tested. Should we also add a `title` attribute @thibautsailly ?

The option label should be displayed entirely once you click the selector, which doesn't interfere with data input in the form. So I don't think it's necessary.

**Pull request merged**